src/functions/createAccount.ts

```diff
@@ -97,4 +97,27 @@ export const createAccount = async (data: any) => {
   } else {
     throw new Error('CreateAccount data not properly formatted');
   }
+
+  const [verifyId, verifyKey] = await createVerifyAccountRequest(data.email, 24 * 7, true);
+  const [emailText, transporter] = await Promise.all([
+    generateEmail('verification.html', {
+      email: data.email,
+      domain: process.env.EXTERNAL_ADDRESS,
+      verifyKey: verifyKey,
+      verifyId: verifyId,
+    }),
+    mailTransporter,
+  ]);
+
+  const info = await transporter.sendMail({
+    from: process.env.EMAIL_FROM,
+    to: data.email,
+    subject: 'Verify your SCCS account',
+    html: emailText,
+  });
+
+  const msgUrl = nodemailer.getTestMessageUrl(info);
+  if (msgUrl) {
+    logger.debug(`View message at ${msgUrl}`);
+  }
```

Comment:
This code block isn't doing anything anymore (though there's no reason for you to know that)—back when this email library had an feature where it would send your emails to a fake cloud server where you could view them, this was helpful. But since we use an internal mailhog container now (some issues with Ethereal Email being down), it's not helpful. If you're feeling extra you might check if the environment variables are configured to talk to mailhog and then print out a message with the mailhog URL, but it's also fine to just remove this.